### DISCUSSION TITLE
Dropdown weird stuff 0.25.1

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#4509] [Q&A]
### Question

Good afternoon. Today, I update my versión of flet, to 0.23 to 0.25.1, and now, the hint text of the dropdown not appear correctly. I dont move anything, but how can I fix this? Thanks

With version 0.21
![image](https://github.com/user-attachments/assets/ef7e48ec-901f-4ed6-be08-578b2e85179b)

With version 0.25.1
![image](https://github.com/user-attachments/assets/a214192e-cb9a-4be1-b360-e938d7c79e39)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Can you do a simple app reproducing the issue please?

**Reply by ingdesarr1** [reply]
```
import flet as ft


def main(page:ft.Page):
    page.title = "Test Dropdown"

    contenedor = ft.Container(
        content=ft.Column(
            controls=[
                ft.Dropdown(
                    options=[
                        ft.dropdown.Option("Valor 1"),
                        ft.dropdown.Option("Valor 2"),
                        ft.dropdown.Option("Valor 3"),
                        ft.dropdown.Option("Valor 4"),
                    ],
                    hint_text="Selector de opciones 1"
                ),
                ft.Dropdown(
                    options=[
                        ft.dropdown.Option("Valor 1"),
                        ft.dropdown.Option("Valor 2"),
                        ft.dropdown.Option("Valor 3"),
                        ft.dropdown.Option("Valor 4"),
                    ],
                    hint_text="Selector de opciones 2"
                ),
            ]
        )
    )

    page.add(contenedor)

    
ft.app(target=main, assets_dir="assets")

```

**Reply by FeodorFitsner** [reply]
This code does not yield any problems:

<img width="825" alt="image" src="https://github.com/user-attachments/assets/99ad6a1d-3342-49d5-a73f-6d4d2056f082">

I see your dropdowns are highly styled/customized. Can you provide a repro reflecting your particular issue?

**Reply by ingdesarr1** [reply]
The problem is not with the selected items, the problem is the hint_text. In the test, I put a hint_text, like "Selector de opciones". If you click the dropdown but not selected any option, in one of the dropdowns, the hint_text is not visible. 

https://github.com/user-attachments/assets/d27eda2b-e440-495a-91fb-da2405d488ca

**Reply by FeodorFitsner** [reply]
Thanks! Looking into that: https://github.com/flet-dev/flet/issues/4511
